oauth2-server/src/main/java/com/clouway/oauth2/AuthCodeAuthorization.java

```diff
@@ -36,12 +38,14 @@ public Response execute(Client client, Request request, DateTime instant) {
 
     Authorization authorization = possibleAuthorization.get();
 
-    Optional<Identity> possibleIdentity = identityFinder.findIdentity(authorization.identityId, GrantType.AUTHORIZATION_CODE, instant);
+    Map<String, String> params  = new CustomParamsParser().parse(request, "code");
```

Comment:
This doesn't look readable as custom is too common. 

Maybe something like: 
```java
Map<String, String> params = Params.parse(request, excluding("code")) 
```
reads better.